docs/source/reference/envvars.rst

```diff
@@ -432,13 +432,14 @@ GPU support
 
 .. envvar:: NUMBA_NPY_RELAXED_STRIDES_CHECKING
 
-   By default Numba device arrays compute their contiguity using relaxed strides
+   By default arrays that inherit from ``numba.misc.dummyarray.Array` (e.g.
```

Comment:
The number of backticks here doesn't add up?

Reply:
Fixed.

Reply:
thanks